frontend/src/recipePage.jsx

```diff
@@ -0,0 +1,89 @@
+import './recipePage.css';
+import React, { useState, useEffect } from 'react';
+
+export default function RecipePage({name, desc, image, ingredients, steps}) {
+  const [recipes, setRecipes] = React.useState();
+  const [ingredientList, setIngredientList] = React.useState('');
+  const [anIngredient, setNewIngredient] = React.useState('');
```

Comment:
this is a nitpick, but it is typically a react convention to keep the variable names consistent with your setters. I would suggest changing anInstruction to newInstruction. This just makes it easier to remember what variables are what once a file grows larger,